mission.sqm

```diff
@@ -358,8 +375,8 @@ class Mission
 {
 	class Intel
 	{
-		briefingName="408 - All Out Warfare v1.61";
```

Comment:
Consolidate versioning to v1.62
Also check in other files (search for string "v1.61" and "v1.70" and "v1.71" in whole project and replace with "v1.62")
Better yet, just revert back to 1.61, we should only change versioning on master, or maybe on develop branch